bundle/Resources/views/Content/content_form.html.twig

```diff
@@ -0,0 +1,23 @@
+{% macro display_form(form) %}
```

Comment:
I don't think it really matters... this template will ultimately be overridable, meaning that it should have a clear path. But I'll keep the option in mind.